ValidAnagram.py

```diff
@@ -0,0 +1,22 @@
+#
+# https://leetcode.com/problems/valid-anagram/description/
+#
+#####################################################################################################################################################
+#                                                            PROBLEM DESCRIPTION                                                                    #
+#####################################################################################################################################################
+#
+# Given two strings s and t, return true if t is an anagram of s, and false otherwise.
+# An Anagram is a word or phrase formed by rearranging the letters of a different word or phrase, typically using all the original letters exactly once.
+
+# Example 1:
+# Input: s = "anagram", t = "nagaram"
+# Output: true
+
+# Example 2:
+# Input: s = "rat", t = "car"
+# Output: false
+#
+#####################################################################################################################################################
+class Solution:
+    def isAnagram(self, s: str, t: str) -> bool:
+        return Counter(s) == Counter(t)
```

Comment:
Good solution thou!

Reply:
But needs Counter imported

Reply:
```
if __name__ == "__main__":
    assert Solution().isAnagram("anagram", "nagaram") == True
    assert Solution().isAnagram("rat", "cat") == False
```

Reply:
As said earlier, we get to test our code solution against the test case in Leet itself. And also the main seems to be added at the back-end with test case data. May be i shall add these assert statements when i work in local code.